src/lib.rs

```diff
@@ -1,7 +1,23 @@
-//! Fast parser of requirement files (txt or in)
+//! Fast parser of Python's requirement files
+//!
+//! `requirements` is a fast and convenient way to parse Python's requirements file. It (will)
+//! supports recursively parsing requirements files and resolving and identifying version
+//! version constraints to satisfy all dependencies.
 //!
 //! # Usage
-//! __Soon…__
+//!
+//! Below is a basic example of how to use this library:
+//!
+//! ```
+//! fn main() {
+//!     let content = "Django==1.11";
```

Comment:
Sorry, since April 1st Django 1 is no longer supported. Let’s have a more future proof example!

Reply:
Fixed.